gr-channels/include/gnuradio/channels/selective_fading_model2.h

```diff
@@ -56,11 +56,10 @@ class CHANNELS_API selective_fading_model2 : virtual public sync_block
      * \param K      Rician factor (ratio of the specular power to the scattered power)
      * \param seed   a random number to seed the noise generators
      * \param delays a vector of values that specify the time delay of each impulse
-     * \param delay_std a vector of values that specify the gaussian step size std of
-     * delays \param delay_maxdev a vector of values that specify the max deviation of
+     * \param delays_std a vector of values that specify the gaussian step size std of
```

Comment:
I do not really like how I did it. Can we do it like [here](https://www.gnuradio.org/doc/doxygen/selective__fading__model_8h_source.html), even though it will be more symbols than 80?

Reply:
You're welcome to try whatever you want & see if it sticks :)

Reply:
I actually prefer minimal changes in a given commit: just fix what's necessary and nothing more. Anything more can be in a separate PR or a separate commit.

Or, do multiple commits in a single PR so that we can follow what's changing & how.

Reply:
Meh it's just a docs fix.